We cannot compare to origin/main git branch because we do not use git repo.

--- filelock-3.12.0/tox.ini.orig
+++ filelock-3.12.0/tox.ini
@@ -30,7 +30,7 @@
       --cov-config=pyproject.toml --no-cov-on-fail --cov-report term-missing:skip-covered --cov-context=test \
       --cov-report html:{envtmpdir}{/}htmlcov --cov-report xml:{toxworkdir}{/}coverage.{envname}.xml \
       tests
-    diff-cover --compare-branch {env:DIFF_AGAINST:origin/main} {toxworkdir}{/}coverage.{envname}.xml}
+    }
 
 [testenv:fix]
 description = format the code base to adhere to our styles, and complain about what we cannot do automatically
